server/cpho/jinja2/indicators/list_indicators.jinja2

```diff
@@ -19,12 +19,13 @@
           <th>{{ tdt("Lead Program") }}</th>
           {# <th>{{ tdt('Detailed indicator') }}</th> #}
           {# <th>{{ tdt('Sub Indicator Mesaurement') }}</th> #}
-          <th>{{ tdt("Actions") }}</th>
+          <th width="15%">{{ tdt("Actions") }}</th>
         </tr>
       </thead>
       <tbody>
         {% for indicator in object_list %}
           {% if respects_rule('can_view_indicator', indicator) %}
+            {# <tr onclick="window.open('{{ url('view_indicator', args=[indicator.pk]) }}');"> #}
```

Comment:
Similar question on this and the other commented out lines in this file, are they left around for a purpose or can they be deleted?

Reply:
I left this one around as this makes the entire row clickable but I wasn't sure if it meets accessibility requirements or if its best practice. (but if we ever want that feature we could comment it back in)

But I think its best to get rid of it as I don't think we will ever need it